client/components/Players/PlayerManager.vue

```diff
@@ -84,7 +84,7 @@
                       <v-btn
                         v-if="supportedFeatures.pictureInPicture"
                         icon
-                        disabled
+                        @click="togglePictureInPicture"
                       >
                         <v-icon> mdi-picture-in-picture-bottom-right </v-icon>
```

Comment:
If you could remove the trailing spaces in the meantime :p

Reply:
Either side of the `v-icon`?